android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -94,6 +96,14 @@ open class MainActivity : ReactActivity() {
         }
     }
 
+    private fun getFileName(uri: Uri, cr: ContentResolver): String? {
```

Comment:
> Perhaps a good name would be `HandleSendIntent.kt`

That's already taken by the activity responsible for receiving `sendIntent` :smile:

I have moved it to a file called `sharing/SharingHelper.kt`, similar to `NotificationHelper.kt`

I have done it all in 1 commit since, there was just 1 parameter (application) to add and nothing much changed, I have added a descriptive commit to explain that, I hope that is fine.

Reply:
Cool, yeah, that single commit worked well because just a handful of lines in the moved code had to change.